src/HtmlGenerator/HtmlGenerator.csproj

```diff
@@ -52,11 +52,10 @@
     <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.Features" Version="$(NuGetVersionRoslyn)" />
     <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.Workspaces" Version="$(NuGetVersionRoslyn)" />
     <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.MSBuild" Version="$(NuGetVersionRoslyn)" />
-    <PackageReference Include="MSBuild.StructuredLogger" Version="2.2.169" />
-    <PackageReference Include="GuiLabs.Language.Xml" Version="1.2.88" />
-    <PackageReference Include="Microsoft.VisualStudio.Language.Intellisense" Version="16.10.230" />

```

Comment:
this was the only dependency requiring us to target net472, we were only using two enums from this libraries, I just copied these enums into `SymbolIdService`